packages/bootstrap-4/src/AddButton/AddButton.tsx

```diff
@@ -5,7 +5,7 @@ import Button from "react-bootstrap/Button";
 import { BsPlus } from "react-icons/bs";
 
 const AddButton: React.FC<AddButtonProps> = props => (
-  <Button {...props} color="primary" style={{width: "100%"}} className="ml-1">
+  <Button {...props} block>
```

Comment:
Why change this?

Reply:
Primary is the default and for full width buttons using `block` is the idiomatic way.